pkg/client/unversioned/kubelet.go

```diff
@@ -39,31 +40,20 @@ type HTTPKubeletClient struct {
 }
 
 func MakeTransport(config *KubeletConfig) (http.RoundTripper, error) {
-	cfg := &Config{TLSClientConfig: config.TLSClientConfig}
-	if config.EnableHttps {
-		hasCA := len(config.CAFile) > 0 || len(config.CAData) > 0
-		if !hasCA {
-			cfg.Insecure = true
```

Comment:
I'll comment there as well, but that method should be calling `hasCA`, not `UsesTLS`